channels/rdpdr/client/rdpdr_main.c

```diff
@@ -1254,7 +1253,7 @@ static UINT dummy_irp_response(rdpdrPlugin* rdpdr, wStream* s)
 	Stream_Write_UINT16(output, PAKID_CORE_DEVICE_IOCOMPLETION); /* PacketId (2 bytes) */
 	Stream_Write_UINT32(output, DeviceId);                       /* DeviceId (4 bytes) */
 	Stream_Write_UINT32(output, CompletionId);                   /* CompletionId (4 bytes) */
-	Stream_Write_INT32(output, STATUS_UNSUCCESSFUL);             /* IoStatus (4 bytes) */
+	Stream_Write_UINT32(output, STATUS_UNSUCCESSFUL);            /* IoStatus (4 bytes) */
```

Comment:
`STATUS_UNSUCCESSFUL` is a `NTSTATUS` so this needs a cast when converting to unsigned.

Reply:
Fixed